### PR TITLE
Add AVX to Troubleshooting

### PR DESCRIPTION
**Pull request by pdmack**
Just pops up in so many weird ways. Time to be more explicit about it IMHO.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/929)

### CODE REVIEW

**Comment by pdmack** [comment]
@jlewi quick-hitter to pad your stats....

File: user_guide.md [hunk 1]
```diff
@@ -550,6 +551,14 @@ tmpfs                                                           15444244       0
 
 ## Troubleshooting
 
+### TensorFlow and AVX
+There are some instances where you may encounter a TensorFlow-related Python installation or a pod launch issue that results in a SIGILL (illegal instruction core dump). Kubeflow uses the pre-built binaries from the TensorFlow project which, beginning with version 1.6, are compiled to make use of the [AVX](https://en.wikipedia.org/wiki/Advanced_Vector_Extensions) CPU instruction. This is a recent feature and your CPU might not support it. Check the host environment for your node to determine whether it has this support:
```
Comment (ankushagarwal) [inline user_guide.md]: SIGILL typo

Comment (pdmack) [reply user_guide.md]: What's the typo?

Comment (ankushagarwal) [reply user_guide.md]: My bad. I thought it was supposed to be SIGKILL

Comment (pdmack) [reply user_guide.md]: 👍

**Comment by pdmack** [comment]
@jlewi can we slide this in?

**Comment by ankushagarwal** [comment]
Can you rebase? There is a problem with the old tests.

**Pull request merged**